[IMP] crm: update win loss report documentation for Odoo 17 (#6746)
odoo/documentation: content/applications/sales/crm/performance/win_loss.rst

   :align: center
   :alt: View of CRM page showing the Reporting tab with the Pipeline view highlighted.

The :guilabel:`Pipeline` dashboard can be customized with various filters and group by options.
Click the dropdown menu in the :guilabel:`Search...` bar, and consider the following common
selections:

#. Filters: :guilabel:`Won`.
#. Filters: :guilabel:`Lost`.

   - Optional: Remove or adjust the :guilabel:`Created On` filter to reflect the last 30 days, the
     last quarter. By default, it reflects the entire year.

   - Optional: Show only leads that have reached the last stage of the sales pipeline (*Closed*) by
     adding a custom filter for :menuselection:`Last Stage Update is >= DATE`.
#. Group By: :guilabel:`Salesperson`.
#. Group By: :menuselection:`Add Custom Group --> Active`.

   - Optional: Group by :guilabel:`Sales Team` or :guilabel:`Company` instead of by
     :guilabel:`Salesperson`.

.. image:: win_loss/search-panel-filters-and-group-by-options.png
   :align: center
   :alt: View of CRM page showing the Search Panel open and the Won and Lost filters highlighted.

After making the above selections, the :guilabel:`Search Panel` should read:

> :guilabel:`Created On: YYYY`, :guilabel:`Won or Lost`, :guilabel:`Salesperson > Active`. If so, the

Author: I don't know if we need to change this one.

When I open CRM in 17.0, I land at the Kanban view. I then open the Reporting menu and click Pipeline. That lands me at the Pipeline Analysis page that already has a bar graph open.

When I open the search bar dropdown and select the filters/group bys, that automatically changes the bar graph to reflect those selections. So really, all they need to do is click off the search bar dropdown (to make it disappear) and then they are viewing the bar graph by default.

I callout the chart icons later because I think they're worth knowing about as additional views, but they aren't relevant for just pulling the base report. Thoughts?

graph now displays the *Count* of wins and losses for each salesperson. Hover over the chart to see
the percentage of wins (true) or losses (false). Click on a section of the chart to open a list of

> those accounts.

Author: I did not realize you could break up the text after :guilabel: but I guess that makes sense as long as you have the backticks.

.. image:: win_loss/win-loss-ratio-bar-chart.png
   :align: center
   :alt: View of Win/Loss Ratio report as a bar chart.

Additional measurements and views
=================================

By default, the win/loss report measures the total *Count* of leads as a *Bar Chart*, but there are
many options for measures and views.

> Go to :menuselection:`CRM --> Reporting --> Pipeline`. Select any desired filters and group by

Reviewer: In line 9, in the navigation to CRM, there's an "app" that follows the app name
> :menuselection:`CRM app --> Reporting --> Pipeline`

I think we should keep this navigational instruction (:menuselection:`[app name] --> [heading] --> [click the option]`) consistent throughout the document. So, we could either remove the word "app" from line 9, or add it in here.

```suggestion
Go to :menuselection:`CRM app --> Reporting --> Pipeline`. Select any desired filters and group by
```
Friendly reminder that there's another navigational instruction like this one on line 72!

Author: Good catch! Thank you. I'll fix the one on line 72 as well.

options for the report, and click the :guilabel:`Measures` button to select other measurement
options such as :guilabel:`Expected Revenue`.

Click the :guilabel:`Chart Icons` along the top to switch the graph type from a Bar Chart to a
*Line Chart* or *Pie Chart*.

Additionally, the report can be changed from a *Graph view* to a *Pivot view*, *Cohort view*, or
*List view*.

.. image:: win_loss/measure-button-menu-pie-chart-view-and-graph-type-button.png
   :align: center
   :alt: The Measures dropdown menu and pie chart view and pivot view.

Add a win/loss report to dashboards
===================================

After configuring the views and measurements, a win/loss report can be saved to a spreadsheet and/or
dashboard for easier viewing in the future.

Go to :menuselection:`CRM --> Reporting --> Pipeline`. Select any desired filters and group by

> options for the report. Now, click on the :guilabel:`Pivot view` button to the right of the Search

Author: I like both of these changes. Agree that we should only use guilabel with the actual text that shows up when you hover over the button/icon.

   - Optional: Show only leads that have reached the last stage of the sales pipeline (*Closed*) by
     adding a custom filter for :menuselection:`Last Stage Update is >= DATE`.
#. Group By: :guilabel:`Salesperson`.
#. Group By: :menuselection:`Add Custom Group --> Active`.

   - Optional: Group by :guilabel:`Sales Team` or :guilabel:`Company` instead of by
     :guilabel:`Salesperson`.

.. image:: win_loss/search-panel-filters-and-group-by-options.png
   :align: center
   :alt: View of CRM page showing the Search Panel open and the Won and Lost filters highlighted.

After making the above selections, the :guilabel:`Search Panel` should read: :guilabel:
`Created On:YYYY`, :guilabel:`Won or Lost`, :guilabel:`Salesperson > Active`. If so, the graph now
displays the *Count* of wins and losses for each salesperson. Hover over the chart to see the
percentage of wins (true) or losses (false). Click on a section of the chart to open a list of those
accounts.

.. image:: win_loss/win-loss-ratio-bar-chart.png
   :align: center
   :alt: View of Win/Loss Ratio report as a bar chart.

Additional measurements and views
=================================

> By default, the win/loss report measures the total *Count* of leads as a Bar Chart, but there are

Reviewer 2: ```suggestion
By default, the win/loss report measures the total :guilabel:`Count` of leads as a Bar Chart, but there are
```
I am erring on the side of caution adding a guilabel here, I will defer to Sam but I think since it refers to an interactive element on the screen it should probably have one? (I am happy to be corrected 😄)

==================
Win/Loss reporting
==================

A key performance indicator available in the *Pipeline Analysis* feature of the *CRM app* is the
win/loss ratio, a metric of the leads/opportunities marked Won or Lost over a period of
time. By calculating "opportunities won" over "opportunities lost", teams can identify what methods
and/or individuals are efficiently converting leads into sales.

.. math::
   \begin{equation}
   Win/Loss Ratio = \frac{Opportunities Won}{Opportunities Lost}

>    \end{equation}

Reviewer 3: Is it necessary? No...
But is it math? YES 🔥 

My vote is we keep it 😁 
[!becauseMath](https://media.giphy.com/media/l2YWF00ZX8wOs0p0s/giphy.gif)

   heading, click :guilabel:`Add Custom Filter`.
3. In the resulting pop-up menu, click the current filter (by default, it is set to
   :guilabel:`Country`) and select :guilabel:`Active` from the drop-down. The rule should now read
   "`Active` `is` `set`".
4. Click :guilabel:`New Rule`, and once more change the default filter to :guilabel:`Active`. Click
   on the last part of this rule and change it to read "`Active` `is` `not set`."
5. Click :guilabel:`Add`.

.. image:: win_loss/add-custom-active-filter.png
   :align: center
   :alt: The Add Custom Filter menu showing two rules: (1) Active is set, and (2) Active is not set.

Click off the *Search* menu to reveal the report, which now displays the total :guilabel:`Count` of
leads, whether won or lost, grouped by their stage in the CRM pipeline. Hover over a section of the
report to see the number of leads in that stage.

.. image:: win_loss/basic-win-loss-report.png
   :align: center
   :alt: A basic win/loss report showing all leads whether won or lost grouped by stage.

Advanced filters and groupings
------------------------------

> The *Pipeline Analysis* can be customized with various filters and group by options. After

Reviewer 4: At this point, the reader is expected to be on the Pipeline Analysis page, so the use of GUI here is required -- because that title is actually on the screen

> creating a basic win/loss report, click on the :guilabel:`Search bar` and consider adding advanced

Reviewer 4: I believe they would be clicking on a down arrow icon beside the search bar to reveal the drop-down menu of options.

filters and groupings to customize the report for different purposes. Some useful options
include:

- **Created On**: Adjusting this filter to a different period of time such as the last 30 days
  or the last quarter can provide more timely results.
- **Add Custom Filter**: Clicking :guilabel:`Add Custom Filter` and scrolling through the numerous
  options in the drop-down opens up additional search criteria like :guilabel:`Last Stage Update` or
  :guilabel:`Lost Reason`.
- **Multiple Groupings**: Adding multiple *Group By* selections can split results into more
  relevant and manageable chunks.

  - Adding :guilabel:`Salesperson` or :guilabel:`Sales Team` breaks up the total count of leads in
    each :guilabel:`Stage` by who they were assigned to.
  - Adding :guilabel:`Medium` or :guilabel:`Source` can reveal what marketing avenues generate more
    sales.
- **Add Custom Group > Active**: Clicking :menuselection:`Add Custom Group --> Active` separates the
  results in each stage into *Won (true)* or *Lost (false)*. This can show at what stage leads are

>   won and lost.

Reviewer 4: I'd suggest expanding this portion to touch on every single option available. Our doc readers prefer over-explanation. You can do this in a simple list format.

Again, I think this portion should appear above, where you introduce the entire page to the reader...

But I'd probably go with something like...

Filters
-------

The :guilabel:`Filters` section allows users to add custom filters to the results on the
:guilabel:`Pipeline Analysis` page.

- :guilabel:`My Pipeline`: only shows personal leads/opportunities.
- :guilabel:`Opportunities`: only shows opportunities.
- :guilabel:`Leads`: only shows leads.

...and so on...

and then you can start another section for 'Group By' and then 'Favorites'

doing so makes things super-organized and easy to reference

.. image:: win_loss/search-panel-filters-and-group-by-options.png
   :align: center
   :alt: The Search menu open and the Won and Lost filters highlighted.

Additional measurements and views
---------------------------------

In addition to filters and group by options, the *Pipeline Analysis* can also display different
measurements, chart types, and :ref:`report views <reporting/views>`. Need to know something other
than the total count of leads? Change the measurement to something more valuable. Want to see the

> report as a spreadsheet? Switch to the :guilabel:`Pivot View`.

Reviewer 4: I appreciate the personality and personability of this type of phrasing, but documentation is purely meant to instruct and guide the reader along the path towards usability. We don't really need to "suggest" or encourage them to "try" different views/elements. We're kinda just meant to be as sterile as possible with documentation - that way, they can quickly get the information they need.

in essence docs are...

Title of thing you want to know about
- brief explanation of thing you want to know about, and how Odoo does it.

Next heading of section to show how to set up for thing you want to know about
- here's how to set up Odoo to do the thing you want to know about

Page of thing you want to know about
- here's how to get to the page where you can do the thing you want to know about. 
- here's what everything means on the page where you can do the thing you want to know about.
- here are all the various view and filter options on the page where you can do the thing you want to know about.

...and so on...

It's just an in-depth, super informative, step-by-step instruction manual for the reader. It's our job to make sure we've left no stone unturned with our explanations. Doing so ensures there's little to no confusion, and they can feel adequately taught throughout their documentation experience

:guilabel:`Days to Close`.

.. image:: win_loss/measure-button-menu.png
   :align: center
   :alt: The Measures drop-down menu showing Count, Expected Revenue, and other options.

In addition to the standard :guilabel:`Bar Chart`, the graph icons along the top of the report can
switch the visual to a :guilabel:`Line Chart` or :guilabel:`Pie Chart`. The :guilabel:`Stacked` icon
further separates the results in a :guilabel:`Bar Chart` or :guilabel:`Line Chart`.

.. image:: win_loss/graph-icons.png
   :align: center
   :alt: The graph icons along the top of the report change how the results are displayed visually.

Finally, the :ref:`four icons in the top-right <reporting/views>` provide more detailed views of the
numbers and records that make up the report. Instead of the default :guilabel:`Graph View`, try
clicking the :guilabel:`Pivot View` to see the report in a table format, or :guilabel:`List View`
to see a single list of every record in the report.

.. image:: win_loss/pivot-and-list-views.png
   :align: center
   :alt: The same win/loss report displays different information when using Pivot View or List View.

Win/Loss in pivot view

> ======================

Reviewer 4: This section may be better served above, if you take on the approach suggested -- wherein you break down every element of the Pipeline Analysis page (including all the different view options)

Reviewer 4: And the following portion about inserting it into a spreadsheet would be a great addition, too

> After creating a basic win/loss report in the *CRM app's Pipeline Analysis*, consider

Reviewer 4: *CRM* app's :guilabel:`Pipeline Analysis` page...

In this instance, the words 'Pipeline Analysis' appear on the U/I...before, it was mentioned in the intro -- but at that point, the reader was just being introduced to the concept, and wasn't "officially" looking at a page with that title -- hence the use of italics was deemed okay.

:ref:`selecting Pivot View <reporting/views>` to see the data in a table format. By default,

> :guilabel:`Pivot View` groups basic win/loss reports by :guilabel:`Stage` and measures

Reviewer 4: since the words 'Pivot View' are immediately seen over the button for it, there's no need to use a GUI in this instance. Same goes for 'Stage' and 'Count' -- but 'Expected Revenue' *DOES* need a GUI because you can see it on the UI/it's present on the screen

- :guilabel:`Stacked`: when selected, the results of each stage of the graph are stacked on top of
  each other. When not selected, the results in each stage are shown as individual bars.
- :guilabel:`Descending`: re-orders the stages in the graph in descending order from left-to-right.
  Click the icon a second time to deselect it. Depending on the search criteria, this option may not
  be available.
- :guilabel:`Ascending`: re-orders the stages in the graph in ascending order from left-to-right.
  Click the icon a second time to deselect it. Depending on the search criteria, this option may not
  be available.

With the pivot view selected, the following options are available:

- :guilabel:`Flip Axis`: flips the X and Y axis for the entire table.
- :guilabel:`Expand All`: when additional groupings are selected using the :guilabel:`➕ (plus
  sign)` icons, this button opens those groupings under every row.
- :guilabel:`Download xlsx`: downloads the table as an Excel file.

.. _win_loss/search:

Search options
--------------

The :guilabel:`Pipeline Analysis` page can be customized with various filters and grouping options.

> To add new search criteria, type the desired criteria into the search bar, or click the :guilabel:`⬇️
> (down arrow)` icon, next to the search bar, to open a drop-down menu of all options. See the
> sections below for more information on what each option does.

Reviewer 3: ```suggestion
To add new search criteria, type the desired criteria into the search bar, or click the
:guilabel:`⬇️ (down arrow)` icon, next to the search bar, to open a drop-down menu of all options.
See the sections below for more information on what each option does.
```

   determines the relationship of the first and third fields, and is usually an **is** or **is not**
   statement, but can also be **greater than or less than** statements, and more.
6. Finally, click the third field, and select an option from the drop-down menu. The third field
   determines the secondary subject of the rule.
7. With all three fields selected, the rule is complete.

   - **To add more rules:** click :guilabel:`New Rule` and repeat steps 4-7, as needed.
   - **To delete a rule:** click the :guilabel:`🗑️ (trash)` icon to the right of the rule.
   - **To duplicate an existing rule:** click the :guilabel:`➕ (plus sign)` icon to the right of
     the rule.
   - **To create more complex rules:** click the :guilabel:`Add branch` icon to the right of the
     rule. This adds another modifier below the rule for adding an "all of" or "any of" statement.

.. image:: win_loss/custom-filter-add-branch.png
   :align: center
   :alt: The add branch feature allows the creation of more complex all or any statements for rules.

8. Once all rules have been added, click :guilabel:`Add` to add the custom filter to the search
   criteria.

>    - **To remove a custom filter:** click the :guilabel:`✖️ (x)` icon beside the filter in the search
>      bar.

Reviewer 3: ```suggestion
   - **To remove a custom filter:** click the :guilabel:`✖️ (x)` icon beside the filter in the
     search bar.
```

below.

.. _win_loss/win_loss:

Win/Loss reports
----------------

Win/Loss is a calculation of active or previously active leads in a pipeline that were either marked
as **Won** or **Lost** over a specific period of time. By calculating opportunities won over
opportunities lost, teams can clarify key performance indicators (KPIs) that are converting leads
into sales, such as specific teams or team members, certain marketing mediums or campaigns, and so
on.

.. math::
   \begin{equation}
   Win/Loss Ratio = \frac{Opportunities Won}{Opportunities Lost}
   \end{equation}

A win/loss report filters the leads from the past year, whether won or lost, and groups the results
by their stage in the pipeline. Creating this report requires a custom filter, and grouping the
results by :guilabel:`Stage`.

.. image:: win_loss/search-criteria-for-basic-win-loss.png
   :align: center
   :alt: The search criteria for win/loss reports is Created On, Stage, and Active is in true false.

Follow the steps below to create a win/loss report:

1. Navigate to :menuselection:`CRM app --> Reporting --> Pipeline`.

> 2. On the :guilabel:`Pipeline Analysis` page, click the :guilabel:`⬇️ (down arrow)` icon, next to the
>    search bar, to open a drop-down menu of filters and groupings.

Reviewer 3: ```suggestion
2. On the :guilabel:`Pipeline Analysis` page, click the :guilabel:`⬇️ (down arrow)` icon, next to
   the search bar, to open a drop-down menu of filters and groupings.
```

=================
Pipeline Analysis
=================

The *CRM* app manages the sales pipeline as leads/opportunities move from stage to stage,

> origination to sale (Won) or archival (Lost).

Reviewer 3: either are fine.
this feedback is optional.

After organizing the pipeline, use the search options and reports in the *Pipeline Analysis* page
to gain insight into the effectiveness of the pipeline and its users.

> To open the *Pipeline Analysis*, go to :menuselection:`CRM app --> Reporting --> Pipeline`.

Reviewer 3: The feedback here @MaxKeilOdoo is specificity of language.

We don't *open* the page, per se, we *access* it by navigating to it in Odoo.
It's a super nitpick but I agree with KC's suggestion here (nitpicks like this are also a sign you're almost done with the doc, fwiw 😉)

.. image:: win_loss/reporting-tab-and-pipeline-view.png
   :align: center
   :alt: Open the CRM app and click on the Reporting tab along the top, then click Pipeline.

.. _win_loss/pipeline:

Navigating the analysis page

> ============================

Reviewer 3: agree.
also try to use simple present instead of continuous (navigate > navigating) whenever possible 👍

Upon opening the :guilabel:`Pipeline Analysis` page, a graph of the opportunities (active leads)
from the past year automatically populates. The bars represent the number of opportunities in each

> stage of the sales pipeline, color-coded to show the month the opportunity reached that stage.

Reviewer 3: I think the edit here was just specificity again. What type of graph? A *bar* graph.

.. image:: win_loss/pipeline-analysis-page.png
   :align: center
   :alt: The default state of the Pipeline Analysis page is a graph, with many options to change it.

The interactive elements of the :guilabel:`Pipeline Analysis` page manipulate the graph to report
different metrics in several views. From left-to-right, top-to-bottom, the elements include:

- :guilabel:`Actions`: The :guilabel:`gear-shaped icon` next to :guilabel:`Pipeline Analysis` opens
  a menu with options for inserting/linking the graph in a *Knowledge* article, adding the graph to
  :guilabel:`My dashboard` in the *Dashboards* app, or linking the graph in a spreadsheet in the
  *Documents* app. (See :ref:`Saving and sharing reports <win_loss/save_reports>` for more
  information)
- :guilabel:`Search bar`: Shows the filters and groupings currently being applied to the graph. To
  add new filters/groups, type them into the search bar or click the :guilabel:`down arrow` at the
  end of the bar to open a menu of options. (See :ref:`Search Options <win_loss/search>` for more

>   information)

Reviewer 3: Can still use the emoji ⬇️ I think.

Also depending on how it's rendered in Odoo, other variations of the downward arrow are ok like these more basic text symbols which are universally supported: ↓ or ▼, for example.

- :guilabel:`Graph`: Switches the analysis to Graph View. By default, this is selected.  (See
  :ref:`View Options <win_loss/view>` for more information)
- :guilabel:`Pivot`: Switches the analysis to Pivot View, which displays the results in a table.
- :guilabel:`Cohort`: Switches the analysis to Cohort View, which displays the results by their
  :guilabel:`Created on` week and :guilabel:`Closed Date` week.

> - :guilabel:`List`: Switches the analysis to List View, which displays the results in a single list.

Reviewer 3: Beginning with a navigational queue on the UI + lowercase first letter after colon ☝️

      drop-down menu with options to insert/link to the :guilabel:`Cohort View` in the *Knowledge*
      app, or add it to a dashboard or spreadsheet.

      The :guilabel:`Cohort View` can also be downloaded as an Excel file by clicking the
      :guilabel:`Download` icon in the top-left of the page.

   .. tab:: List View
      The List View displays a single list of all leads/opportunities matching the search criteria.
      This view is useful for reviewing and digging deeper on many records at once.

      Clicking on a lead/opportunity opens the record for closer review.

      .. image:: win_loss/list-view.png
         :align: center
         :alt: The List View displays a single list of all records matching the search criteria.

      Additional details for each record such as :guilabel:`Country`, :guilabel:`Medium`, and more
      can be added to the list by clicking the :guilabel:`Filters` icon in the top-right of the
      list.

      .. image:: win_loss/list-view-columns.png
         :align: center
         :alt: More details can be added to the list by clicking the Filters icon in the top-right.

>       Clicking the gear-shaped :guilabel:`Actions` icon opens a drop-down menu with options for the
>       following:

Reviewer 4: ```suggestion
      Clicking the :guilabel:`⚙️ (gear)` icon opens the Actions drop-down menu, with options for the
      following:
```

>       - :guilabel:`Import records`: Opens a page for uploading a spreadsheet of data, as well as a
>         template spreadsheet for easily formatting that data.
>       - :guilabel:`Export All`: Downloads the list as an xlsx file for Excel.
>       - :guilabel:`Knowledge`: Inserts a view of or link to the list in an article in the
>         *Knowledge* app.
>       - :guilabel:`Dashboard`: Adds the list to :guilabel:`My Dashboard` in the *Dashboards* app.
>       - :guilabel:`Spreadsheet`: Links to or inserts the list in a spreadsheet in the *Documents*
>         app.

Reviewer 4: ```suggestion
      - :guilabel:`Import records`: opens a page for uploading a spreadsheet of data, as well as a
        template spreadsheet to easily format that data.
      - :guilabel:`Export All`: downloads the list as an xlsx file for Excel.
      - :guilabel:`Knowledge`: inserts a view of, or link to, the list in an article in the
        *Knowledge* app.
      - :guilabel:`Dashboard`: adds the list to *My Dashboard* in the *Dashboards* app.
      - :guilabel:`Spreadsheet`: links to, or inserts, the list in a spreadsheet in the *Documents*
        app.

```

      .. note::

>          At the List View, clicking :guilabel:`New` closes the list and opens the *New Quotation*
>          page. Clicking :guilabel:`Generate Leads` opens a pop-up for lead generation. Neither
>          feature is intended to manipulate the List View.

Reviewer 4: ```suggestion
         On the list View, clicking :guilabel:`New` closes the list, and opens the *New Quotation*
         page. Clicking :guilabel:`Generate Leads` opens a pop-up window for lead generation.
         Neither feature is intended to manipulate the list View.
```

.. _win_loss/reports:

> Creating reports
> ================

Reviewer 4: ```suggestion
Create reports
==============
```

> After understanding how to :ref:`navigate the analysis page <win_loss/pipeline>`, the
> :guilabel:`Pipeline Analysis` can be used to create and share different reports. Between the
> pre-made options and custom filter/groupings, almost any combination is possible.

Reviewer 4: ```suggestion
After understanding how to :ref:`navigate the pipeline analysis page <win_loss/pipeline>`, the
:guilabel:`Pipeline Analysis` page can be used to create and share different reports. Between the
pre-made options and custom filter/groupings, almost any combination is possible.
```

Once created, reports can be :ref:`saved to favorites, shared with other users, and added to
knowledge articles, dashboards, and spreadsheets <win_loss/save_reports>`.

> A few common reports that can be created using the :guilabel:`Pipeline Analysis` are detailed below.

Reviewer 4: ```suggestion
A few common reports that can be created using the :guilabel:`Pipeline Analysis` page are detailed
below.
```

.. _win_loss/win_loss:

Win/Loss reports
----------------

> Win/Loss is a metric of the leads/opportunities marked Won or Lost over a period of time. By
> calculating "opportunities won" over "opportunities lost," teams can identify what methods and/or
> individuals are effectively converting leads into sales.

Reviewer 4: ```suggestion
Win/Loss is a metric of the leads/opportunities marked "Won" or "Lost" over a period of time. By
calculating "opportunities won" over "opportunities lost," teams can identify what methods and/or
individuals are effectively converting leads into sales.
```

> A win/loss report filters **the leads/opportunities from the past year, whether won or lost, and
> groups the results by their stage in the pipeline**. Creating this report requires a custom filter
> and grouping the results by :guilabel:`Stage`.

Reviewer 4: ```suggestion
A win/loss report filters the leads/opportunities from the past year, whether "Won" or "Lost," and
groups the results by their stage in the pipeline. Creating this report requires a custom filter,
and grouping the results by :guilabel:`Stage`.
```

.. image:: win_loss/search-criteria-for-basic-win-loss.png
   :align: center
   :alt: The search criteria for win/loss reports is Created On, Stage, and Active is in true false.

Follow the steps below to create a win/loss report:

1. Navigate to :menuselection:`CRM app --> Reporting --> Pipeline`.

> 2. At the :guilabel:`Pipeline Analysis` page, click on the :guilabel:`down arrow` next to the search
>    bar to open a menu of filters and groupings.

Reviewer 4: ```suggestion
2. On the :guilabel:`Pipeline Analysis` page, click on the :guilabel:`down arrow` icon, next to the
   search bar, to open a drop-down menu of filters and groupings.
```

   .. image:: win_loss/filters-for-basic-win-loss-report.png
      :align: center
      :alt: The Search menu containing the filters for a basic win/loss report.

3. In drop-down menu that appears, under the :guilabel:`Group By` heading, click :guilabel:`Stage`.
4. Under the :guilabel:`Filters` heading, click :guilabel:`Add Custom Filter` to open another pop-up
   menu.
5. In the :guilabel:`Add Custom Filter` pop-up menu, click on the first field in the
   :guilabel:`Match any of the following rules:` section. By default, this field displays
   :guilabel:`Country`.
6. Clicking that first field reveals a sub-menu with numerous options to choose from. From this
   sub-menu, locate and select the :guilabel:`Active` option. Doing so, auto-populates the remaining

>    fields. The first field reads: :guilabel:`Active`. The second field reads: :guilabel:`is`. And
>    lastly, the third field reads: :guilabel:`set`. In total, the rule reads: :guilabel:`Active is
>    set`.

Reviewer 4: ```suggestion
   fields.
   
   The first field reads: :guilabel:`Active`. The second field reads: :guilabel:`is`. And lastly,
   the third field reads: :guilabel:`set`.
   
   In total, the rule reads: :guilabel:`Active is set`.
```

7. Click :guilabel:`New Rule`, change the first field to :guilabel:`Active`, and the last field to
   :guilabel:`not set`. In total, the rule reads :guilabel:`Active is not set`.
8. Click :guilabel:`Add`.

.. image:: win_loss/add-custom-active-filter.png
   :align: center
   :alt: The Add Custom Filter menu showing two rules: (1) Active is set, and (2) Active is not set.

> The report now displays the total :guilabel:`Count` of leads, whether won or lost, grouped by their
> stage in the CRM pipeline. Hover over a section of the report to see the number of leads in that
> stage.

Reviewer 4: ```suggestion
The report now displays the total :guilabel:`Count` of leads, whether "Won" or "Lost," grouped by
their stage in the CRM pipeline. Hover over a section of the report to see the number of leads in
that stage.
```

.. image:: win_loss/basic-win-loss-report.png
   :align: center
   :alt: A basic win/loss report showing all leads whether won or lost grouped by stage.

> Customizing win/loss reports
> ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Reviewer 4: ```suggestion
Customize win/loss reports
~~~~~~~~~~~~~~~~~~~~~~~~~~
```

After :ref:`creating a win/loss report <win_loss/win_loss>`, consider using the options below to
customize the report for different needs.

.. example::
   A sales manager might group wins and losses by salesperson, or sales team, to see who has the
   best conversion rate. Or, a marketing team might group by sources, or medium, to determine where
   their advertising has been most successful.

.. tabs::

   .. tab:: Filters and groups

>       To add more filters and groups, click the :guilabel:`down arrow` next to the search bar and
>       select one or more options from the drop-down menu.

Reviewer 4: ```suggestion
      To add more filters and groups, click the :guilabel:`down arrow` icon, next to the search bar,
      and select one or more options from the drop-down menu.
```

      Some useful options include:

>       - **Created On**: Adjusting this filter to a different period of time such as the last 30 days
>         or the last quarter can provide more timely results.
>       - **Add Custom Filter**: Clicking :guilabel:`Add Custom Filter` and scrolling through the
>         numerous options in the drop-down opens up additional search criteria like :guilabel:`Last
>         Stage Update` or :guilabel:`Lost Reason`.
>       - **Multiple Groupings**: Adding multiple :guilabel:`Group By` selections can split results
>         into more relevant and manageable chunks.
> 
>         - Adding :guilabel:`Salesperson` or :guilabel:`Sales Team` breaks up the total count of
>           leads in each :guilabel:`Stage` by who they were assigned to.
>         - Adding :guilabel:`Medium` or :guilabel:`Source` can reveal what marketing avenues generate
>           more sales.
>       - **Add Custom Group > Active**: Clicking :menuselection:`Add Custom Group --> Active`
>         separates the results into Won (:guilabel:`true`) or Lost (:guilabel:`false`). This shows
>         at what stage leads are being marked won or lost.

Reviewer 4: ```suggestion
      - :guilabel:`Created on`: adjusting this filter to a different period of time, such as the
        last 30 days, or the last quarter, can provide more timely results.
      - :guilabel:`Add Custom Filter`: clicking this option, and scrolling through the numerous
        options in the drop-down menu, opens up additional search criteria, like :guilabel:`Last
        Stage Update` or :guilabel:`Lost Reason`.
      - :guilabel:`Multiple Groupings`: add multiple :guilabel:`Group By` selections to split
        results into more relevant and manageable chunks.

        - Adding :guilabel:`Salesperson` or :guilabel:`Sales Team` breaks up the total count of
          leads in each :guilabel:`Stage`.
        - Adding :guilabel:`Medium` or :guilabel:`Source` can reveal what marketing avenues generate
          more sales.
      - :guilabel:`Add Custom Group > Active`: Clicking :menuselection:`Add Custom Group --> Active`
        separates the results into "Won" (:guilabel:`true`) or "Lost" (:guilabel:`false`). This
        shows at what stage leads are being marked "Won" or "Lost."
```

         report containing :guilabel:`duplicate group bys`.

         To fix this, replace the :guilabel:`Stage` grouping in the search bar with another option.

   .. tab:: List View

      In :guilabel:`List View`, a win/loss report displays all leads on a single page.

      To add more columns to the list, click the :guilabel:`Filters` icon in the top-right of the
      page, and select options from the resulting drop-down menu. Some useful filters include:

      - **Campaign**: Shows the marketing campaign that originated each lead.
      - **Medium**: Shows the marketing medium (Banner, Direct, Email, Google Adwords, Phone,
        Website, etc.) that originated each lead.
      - **Source**: Shows the source of each lead (Newsletter, Lead Recall, Search Engine, etc.).

      To better organize the list, click the :guilabel:`down arrow` next to the search bar, and
      add more relevant groupings or re-organize the existing ones.

      To re-order the nesting, remove all :guilabel:`Group By` options and re-add them in the
      desired order.

      .. image:: win_loss/win-loss-list-view.png
         :align: center

>          :alt: A win/loss report in List View displays all leads in an easy-to-read list.

Reviewer 4: This section seems repetitive, and I feel like all of this was adequately explained above. I'd suggest removing it.

.. _win_loss/save_reports:

> Saving and sharing reports
> ==========================

Reviewer 4: ```suggestion
Save and share reports
======================
```

After :ref:`creating a report <win_loss/reports>`, the search criteria can be saved, so the report
does not need to be created again in the future. Saved searches automatically update their results
every time the report is opened.

> Additionally, reports can be shared with others or added to spreadsheets/dashboards for greater
> customization and even easier access.

Reviewer 4: ```suggestion
Additionally, reports can be shared with others, or added to spreadsheets/dashboards for greater
customization and easier access.
```

.. tabs::

   .. tab:: Save to Favorites

      To save a report for later:

>       At the :guilabel:`Pipeline Analysis` page, click the :guilabel:`down arrow` next to the search
>       bar, and under the :guilabel:`Favorites` heading, click :guilabel:`Save current search`.

Reviewer 4: ```suggestion
      On the :guilabel:`Pipeline Analysis` page, click the :guilabel:`down arrow` icon, next to the
      search bar, and under the :guilabel:`Favorites` heading, click :guilabel:`Save current
      search`.
```

>       In the drop-down menu that appears, enter a name for the report. By default, the report is
>       named "Pipeline".

Reviewer 4: ```suggestion
      In the drop-down menu that appears, enter a name for the report.
```

>       - Optional: Checking the :guilabel:`Default filter` box sets this report as the default
>         analysis when opening the :guilabel:`Pipeline Analysis` page.
>       - Optional: Checking the :guilabel:`Shared` box makes this report available to other
>         users.

Reviewer 4: ```suggestion
      Checking the :guilabel:`Default filter` box sets this report as the default analysis when the
      :guilabel:`Pipeline Analysis` page is opened. Checking the :guilabel:`Shared` box makes this
      report available to other users.
```

         entire spreadsheet to a dashboard by clicking :menuselection:`File --> Add to dashboard`.

         In the resulting pop-up menu, name the spreadsheet and select a :guilabel:`Dashboard
         Section` to house the report. Once complete, click :guilabel:`Create`.

         Using this method, the spreadsheet can be added to a public dashboard instead of only
         :guilabel:`My Dashboard`.

   .. tab:: Add to a Dashboard

      Adding a report to a dashboard saves it for later and makes it easy to view alongside the rest
      of :guilabel:`My Dashboard`.

      To add a report to :guilabel:`My dashboard`:

      At the :guilabel:`Pipeline Analysis` page, click the gear-shaped :guilabel:`Actions` icon, and
      in the resulting drop-down menu, hover over :guilabel:`Dashboard`.

      Under the :guilabel:`Add to my dashboard` heading, enter a name for the report (by default, it
      is named "Pipeline") and click :guilabel:`Add`.

      To see the saved report, return to the main apps page, and navigate to
      :menuselection:`Dashboards app --> My Dashboard`.

      .. image:: win_loss/add-to-dashboard.png
         :align: center

>          :alt: To access the saved report, open the Dashboard app and click My Dashboard.

Reviewer 4: I believe this section can be deleted, as well -- since you've already covered it in great detail previously in the doc.

=================
Pipeline Analysis
=================

The *CRM* app manages the sales pipeline as leads/opportunities move from stage to stage,
origination to sale (Won) or archival (Lost).

After organizing the pipeline, use the search options and reports available on the *Pipeline
Analysis* page to gain insight into the effectiveness of the pipeline and its users.

To access the *Pipeline Analysis* page, go to :menuselection:`CRM app --> Reporting --> Pipeline`.

.. image:: win_loss/reporting-tab-and-pipeline-view.png
   :align: center
   :alt: Open the CRM app and click on the Reporting tab along the top, then click Pipeline.

.. _win_loss/pipeline:

Navigate the pipeline analysis page
===================================

Upon accessing the :guilabel:`Pipeline Analysis` page, a bar graph showcasing the opportunities
(active leads) from the past year automatically populates. The bars represent the number of
opportunities in each stage of the sales pipeline, color-coded to show the month the opportunity
reached that stage.

> .. image:: win_loss/pipeline-analysis-page.png

Reviewer 3: [screenshot is not in HD](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#screenshots), ideally the width caps to about 933px.

Double check the rest of the folder to make sure images are in HD and aren't too large in dimension. Remove any unnecessary screenshots or any that aren't used any more due to the rewrite.

   statement, but can be "greater than/less than" statements, whether the first field "contains"
   certain elements, and more.

.. image:: win_loss/custom-filter-second-field.png
   :align: center
   :alt: The second field of a custom filter determines the relationship of the subjects.

6. Finally, click the third field, and select an option from the drop-down menu. The third field
   determines the secondary subject of the rule.

.. image:: win_loss/custom-filter-third-field.png
   :align: center
   :alt: The third field of a custom filter determines the secondary subject of the rule.

7. With all three fields selected, the rule is complete.

   - **To add more rules:** click :guilabel:`New Rule` and repeat steps 4-7, as needed.
   - **To delete a rule:** click the :guilabel:`🗑️ (trash)` icon to the right of the rule.
   - **To duplicate an existing rule:** click the :guilabel:`➕ (plus sign)` icon to the right of the
     rule.
   - **To create more complex rules:** click the :guilabel:`Add branch` icon to the right of the
     rule. This adds another modifier below the rule for adding an "all of" or "any of" statement.

> .. image:: win_loss/custom-filter-add-branch.png

Reviewer 3: show a much more completed example to visually drive in the lesson and the value

   :align: center
   :alt: The add branch feature allows the creation of more complex all or any statements for rules.

8. Once all rules have been added, click :guilabel:`Add` to add the custom filter to the search
   criteria.

   - **To remove a custom filter:** click the :guilabel:`✖️ (x)` icon beside the filter in the search
     bar.

**To add a custom group:**

1. On the :guilabel:`Pipeline Analysis` page, click the :guilabel:`down arrow` icon next to the
   search bar.
2. In the drop-down menu that appears, click :guilabel:`Add Custom Group`.
3. Scroll through the options in the drop-down menu, and select one or more groups.

   - **To remove a custom group:** click the :guilabel:`✖️ (x)` icon beside the custom group in the
     search bar.

> .. image:: win_loss/add-custom-group.png

Reviewer 3: don't need

   :align: center
   :alt: The Add Custom Group drop-down menu has many options for additional groupings.

.. _win_loss/measure:

Measurement options
-------------------

By default, the :guilabel:`Pipeline Analysis` page measures the total :guilabel:`Count` of
leads/opportunities that match the search criteria, but can be changed to measure other items of
interest.

> .. image:: win_loss/measure-button-menu.png

Reviewer 3: don't need

  exceeded its Expected Closing date.
- :guilabel:`Expected MRR`: measures the Expected Recurring Revenue of a lead/opportunity.
- :guilabel:`Expected Revenue`: measures the Expected Revenue of a lead/opportunity.
- :guilabel:`Prorated MRR`: measures the Prorated Monthly Recurring Revenue of a lead/opportunity.
- :guilabel:`Prorated Recurring Revenues`: measures the Prorated Recurring Revenues of a
  lead/opportunity.
- :guilabel:`Prorated Revenue`: measures the Prorated Revenue of a lead/opportunity.
- :guilabel:`Recurring Revenues`: measures the Recurring Revenue of a lead/opportunity.
- :guilabel:`Count`: measures the total amount of leads/opportunities that match the search
  criteria. By default, this measurement is selected.

.. _win_loss/view:

View options
------------

After configuring filters, groupings, and measurements, the :guilabel:`Pipeline Analysis` page can
display the data in a variety of ways. By default, the page uses the graph view, but can be changed
to a pivot view, cohort view, or list view.

To change the pipeline to a different view, click one of the four view icons, located in the
top-right of the :guilabel:`Pipeline Analysis` page.

> .. image:: win_loss/pivot-and-list-views.png

Reviewer 3: don't need

        *Knowledge* app.
      - :guilabel:`Dashboard`: adds the list to *My Dashboard* in the *Dashboards* app.
      - :guilabel:`Spreadsheet`: links to, or inserts, the list in a spreadsheet in the *Documents*
        app.

      .. note::
         On the list view, clicking :guilabel:`New` closes the list, and opens the *New Quotation*
         page. Clicking :guilabel:`Generate Leads` opens a pop-up window for lead generation.
         Neither feature is intended to manipulate the list view.

.. _win_loss/reports:

Create reports
==============

After understanding how to :ref:`navigate the pipeline analysis page <win_loss/pipeline>`, the
:guilabel:`Pipeline Analysis` page can be used to create and share different reports. Between the
pre-made options and custom filter/groupings, almost any combination is possible.

Once created, reports can be :ref:`saved to favorites, shared with other users, and/or added to
dashboards and spreadsheets <win_loss/save_reports>`.

A few common reports that can be created using the :guilabel:`Pipeline Analysis` page are detailed
below.

.. _win_loss/win_loss:

Win/Loss reports
----------------

> Win/Loss is a metric of the leads/opportunities marked "Won" or "Lost" over a period of time. By
> calculating "opportunities won" over "opportunities lost," teams can identify what methods and/or
> individuals are effectively converting leads into sales.

Reviewer 3: the punctuation in this section is unnecessary and distracting...
here's how I would simplify the punctuation and expand on the lesson that's trying to be communicated here.

```suggestion
Win/Loss is a calculation of active or previously active leads in a pipeline that were either marked
as **Won** or **Lost** over a specific period of time. By calculating opportunities won over
opportunities lost, teams can identify clarify key performance indicators (KPIs) that are converting
leads into sales, such as specific teams or team members, certain marketing mediums or campaigns,
and so on.
```

.. math::
   \begin{equation}
   Win/Loss Ratio = \frac{Opportunities Won}{Opportunities Lost}
   \end{equation}

A win/loss report filters the leads/opportunities from the past year, whether "Won" or "Lost," and
groups the results by their stage in the pipeline. Creating this report requires a custom filter,

> and grouping the results by :guilabel:`Stage`.

Reviewer 3: again i don't like the quotes (" ") and slash (/) punctuation when it's not necessary.

> ========================

Reviewer: Thanks for changing the branch name to CRM! Reminder to change the name of the PR to 
[IMP] CRM: ... too, but clicking the edit button at the top of the PR 
![image](https://github.com/odoo/documentation/assets/23611711/02ffa5ce-dd64-4ad9-ae7e-ed66a4208af6)

Create a Win/Loss Report
========================

A Win/Loss ratio is the number of leads or opportunities a salesperson, sales team, or company have
won and lost, often represented as a percentage of wins over losses. In the CRM app, the Win/Loss
ratio is available by creating a Win/Loss report.

To create this report, open the CRM app, click the *Reporting* tab, and select the *Pipeline* view.

.. image:: win_loss/reporting-tab-and-pipeline-view.png
    :align: center
    :alt: View of CRM page showing the Reporting tab with the Pipeline view highlighted.

Open the *Search Panel* dropdown, and make the following selections:

#. Filters: *Won*.
#. Filters: *Lost*.

>    - Optional: Removing or adjusting the *Created On* filter to reflect the last 30 days, the last

Author: Change "Removing or adjusting" to "Remove or adjust"
```suggestion
   - Optional: Remove or adjust the *Created On* filter to reflect the last 30 days, the last
```

     adding a custom filter for *Last Stage Update is >= DATE*.
#. Group By: *Salesperson*.
#. Group By: *Add Custom Group --> Active*.

   - Optional: Group by *Sales Team* or *Company* instead of by "Salesperson*.

.. image:: win_loss/search-panel-filters-and-group-by-options.png
    :align: center
    :alt: View of CRM page showing the Search Panel open and the Won and Lost filters highlighted.

The *Search Panel* should read: *Filter: Created On: YYYY, Filter: Won or Lost, Group By:
Salesperson > Active*. If so, it now displays the *count* of wins and losses for each salesperson.
Hover over the chart to see the percentage of wins (true) or losses (false). Click on a
section of the chart to open a list of those accounts.

.. image:: win_loss/win-loss-ratio-bar-chart.png
   :align: center
   :alt: View of Win/Loss Ratio report as a bar chart.

Additional Measurements and Views
=================================
By default, the Win/Loss report measures the total *count* of leads. Click the
*Measures* button to select other options such as *Expected Revenue*.

Click the chart icons along the top to switch the graph type from a Bar Chart to a Line Chart or Pie
Chart.

> Additionally, the report can be changed from a Graph view to a Pivot view, Cohort view, or

Author: So for this one, you're suggesting I italicize the first instance of "Bar Chart... Line Chart... Pie Chart", and then leave the second instances as they are?

List view.

.. image:: win_loss/measure-button-menu-pie-chart-view-and-graph-type-button.png
   :align: center
   :alt: The Measures dropdown menu and pie chart view and pivot view.

Add a Win/Loss Report to Dashboards
===================================
After configuring the views and measurements, a Win/Loss report can be saved to a spreadsheet and/or
dashboard for easier viewing in the future.

From the Win/Loss report, switch to the *Pivot* view. Click the *Measures* button
and select *count*. Click *Insert in Spreadsheet*.

.. image:: win_loss/select-count-and-insert-in-spreadsheet.png
   :align: center
   :alt: Win/Loss report in pivot view showing the Measures dropdown menu.

> Select *Blank Spreadsheet* and click *Confirm*. Name the new spreadsheet,
> and add charts or additional formulas as desired.

Reviewer: Have you ever tried the command `make review`? it helps catch instances where you can fit a few more words onto the line and get it as close to 100 characters as possible. 

To run make review, go to github and copy the file path, in this case
`content/applications/sales/crm/performance/win_loss.rst`
then, REMOVE content/, so u are left with
`applications/sales/crm/performance/win_loss.rst`

Go to terminal, and in the documentation folder, run `make review`
and then, enter the path we just made
When they ask the question about 100 char, just press the Enter button on your keyboard
![image](https://github.com/odoo/documentation/assets/23611711/8d760867-d397-4bef-a5e9-a53166d1c878)

Then, you'll see messages like this:
![image](https://github.com/odoo/documentation/assets/23611711/cb8c36c0-2ed3-45c9-ade9-205872db4fa3)

Saying that in lines 36, 45, 51, 63, 70, and 73, u have an early line break and u can reformat. (For example, shove more words from line 71 onto 70) 
```suggestion
Select *Blank Spreadsheet* and click *Confirm*. Name the new spreadsheet, and add charts or
additional formulas as desired.
```

Once complete, open the *File* menu and select *Add to dashboard*. Name the dashboard,
select a dashboard section, add access groups, and click *Create*.

.. image:: win_loss/add-the-spreadsheet-to-dashboards.png
   :align: center
   :alt: Win/Loss report in a spreadsheet showing the Add to Dashboard option.

> The Win/Loss report is now viewable in the *Dashboards* app. This view automatically updates to

Reviewer: great job remembering  that app names are italicized!
